### DISCUSSION TITLE
Fire TV Omni

### DISCUSSION THREAD

**Discussion by kylebellamy** [projectM-visualizer/projectm#589] [Q&A] [answered]
Just bought this tv on sale and noticed this software on there so I bought it but it does not seem to react to Spotify on the tv. The audio tracks included do not play anything so I'm wondering if this is just something not programmed into the app and if not... Please? lol

**Comment by kblaschke** [comment, marked as answer]
Sadly, the projectM dev team cannot provide support for the Android app, as it's not being written or maintained by us, but someone else (Peter Sperl). Please contact the author directly to get help on your issue.
